bin/resty

```diff
@@ -56,6 +56,7 @@ my $errlog_level = "warn";
 
 GetOptions("c=i",             \(my $conns_num),
            "e=s",             \@src_a,
+           "l=s",             sub { push @src_a, "require('$_[1]')"; },
```

Comment:
Can we use a variable here instead so that we can save this dynamic Perl closure creation overhead when the `-l` option is not actually used?

Reply:
Also, we need to escape any special chars in the `$_[1]` argument to avoid Lua syntax errors when the file name contains funny characters.

Reply:
Extracted it into a subroutine. Hope that is what you had in mind.